### ISSUE TITLE
Inaccuracy of Benefits&Observations #2

### ISSUE THREAD

**Issue by zardus** [ieee-security/ongoing-submission-plan#5]
First off: thank you for having this discussion and planning in such an open space. This is an extremely important change and it looks like the process of making it is off to a really great start!

To continue in that vein, I think that we should be very clear about the realities of paper submission and publication in our field. Namely, the following statement in `Benefits and Observations` is a bit removed from reality:

```
If you measure between completion of the research and publication, the improvement
is even more dramatic, since in the current system, any idea completed after mid-November
must wait up to 12 months just for submission.
```

As we all know, this is not the case. There _are_ other top-tier conferences, and I feel that ignoring their existence in this process will decrease the quality of the final result. IMO, an analogous benefit that could be put in this slot is a discussion on the improvement of the proposed method over the two-week resubmission cycle that occurs currently (where authors have around two weeks between the notifications of one conference and submission deadline of the next).

If you're ok with that, I can make a PR with some proposed wording in that direction.

**Comment by parno** [comment]
Sure, we can add something to clarify that the comment above is specific to Oakland and note that waiting until USENIX is shorter, but typically still a three month delay.  

I agree that we can also note the downsides to the current gap of two weeks between rejection notification and resubmissions at the next conference.  Feel free to create a PR about this.

**Comment by zardus** [comment]
PR opened! Let me know if there's anything I should change or clarify.

**Comment by parno** [comment]
Closed by b20b8124c49f5261458df28e976d7d30a9cfd01a
